CHANGELOG.md

```diff
@@ -75,6 +76,7 @@ All notable changes to this project will be documented in this file.
 - Fix race condition when streaming out deleted statuses ([ThibG](https://github.com/tootsuite/mastodon/pull/10280))
 - Fix performance of admin federation UI by caching account counts ([Gargron](https://github.com/tootsuite/mastodon/pull/10374))
 - Fix JS error on pages that don't define a CSRF token ([hinaloe](https://github.com/tootsuite/mastodon/pull/10383))
+- Fix unreachable domains sometimes deleting accounts in `tootctl accounts cull` ([BenLubar](https://github.com/tootsuite/mastodon/pull/10460))
```

Comment:
Fix `tootctl accounts cull` sometimes removing accounts that are temporarily unreachable